app/Http/Controllers/Api/v1/GameController.php

```diff
@@ -18,13 +18,13 @@ public function index(){
             ->get();
     }
 
-    public function show($id){
+    public function show($id) {
         $game = Game::with('developers', 'user', 'maker', 'screenshots', 'comments')->whereId($id)->first();
 
         return $game;
     }
 
-    public function show_app(){
+    public function show_app() {
```

Comment:
__[code_sniffer]__
* Method name "GameController::show_app" is not in camel caps format
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->